src/components/Application/Navigation.tsx

```diff
@@ -136,6 +177,25 @@ const Navigation: React.FC<NavigationProps> = ({
             {isValidating && <Loader active inline size="tiny" />}
           </Button>
         </div>
+        {isMobile && sectionOptions.length > 1 && (
```

Comment:
Was going to mention about the aligning of buttons when there is a "previous" and "next buttons, but I just noticed it has nothing to do with what you are doing here.  If its an easy fix... else it gets done elsewhere.
![image](https://github.com/openmsupply/conforma-web-app/assets/17306152/c1f082b7-eb91-4d77-b25b-9997457f92b0)

Reply:
<img width="461" alt="Screenshot 2023-11-04 at 12 04 37 AM" src="https://github.com/openmsupply/conforma-web-app/assets/5456533/166c4a68-12b7-4b12-91ad-8f7ebae62300">

Ok, not perfect, but at least it's all flush on the left now. e4731155